packages/builders/scripts/inter-protocol/replace-charter-core.js

```diff
@@ -0,0 +1,54 @@
+import { makeHelpers } from '@agoric/deploy-script-support';
```

Comment:
hm. I wasn't expecting a separate builder script for each issue. I suppose it could work, but it seems kinda awkward.

I guess I was expecting this to use the same builder script from #10164 and just expand the core eval code. What do you think about stacking the PRs?